src/SignalR/common/Shared/MessageBuffer.cs

```diff
@@ -179,7 +179,7 @@ public void Ack(AckMessage ackMessage)
     {
         // TODO: what if ackMessage.SequenceId is larger than last sent message?
```

Comment:
Does this need to be addressed?

Reply:
I don't think it's important. It's more like provide an error for custom clients that implement acking incorrectly.